server/src/metadata/field-metadata/dtos/create-field.input.ts

```diff
@@ -12,24 +12,48 @@ import {
 export class CreateFieldInput {
   @IsString()
   @IsNotEmpty()
-  @Field()
+  @Field({
+    deprecationReason: 'Use name instead',
+    defaultValue: '[deprecated]',
+  })
```

Comment:
@Weiko Je pense qu'on peut hard drop, on a rien de live pour le moment, il faut juste bien prévenir @lucasbordeau

Reply:
@magrinj if I hard drop I believe it will break the FE. Well, we have the FE and the BE on the same repo so I could regenerate the schema on the FE too... I prefer to not mix server and frontend stuff in the same PR in general though 🤔

Reply:
Actually let's do it, I'll do the codegen in this PR since we have the luxury to be able to do that @magrinj